monadic-html/src/main/scala/mhtml/mount.scala

```diff
@@ -94,15 +98,18 @@ object mount {
 
     def setMetadata(m: MetaData, v: Any, config: MountSettings): Unit = {
       val htmlNode = node.asInstanceOf[dom.html.Html]
-      def set(key: String): Unit =
-        if (v == null) htmlNode.removeAttribute(key)
-        else {
+      def set(key: String): Unit = v match {
+        case null | None | false => htmlNode.removeAttribute(key)
```

Comment:
TBH I'm not really sold on this `true`/`false` sementic, what if the user wants to actually print false? Isn't a bit weird to have `1 → "1"`, `1.1 → "1.1"`, `'c' → "c"` but `false → Ø`?